infra/helm/meshdb/templates/meshweb.yaml

```diff
@@ -12,8 +12,9 @@ spec:
       {{- include "meshdb.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
+        admission.datadoghq.com/python-lib.version: v2.11.2
```

Comment:
Not quite latest, but very recent

Reply:
Oh I think I could just pin it to a version tag: https://hub.docker.com/r/datadog/dd-lib-python-init/tags

Same way we do with our python dependencies.

Reply:
```suggestion
        admission.datadoghq.com/python-lib.version: v2.12
```